### ISSUE TITLE
Create function in atomic subpackage to return a particle's electric charge

### ISSUE THREAD

**Issue by namurphy** [PlasmaPy/PlasmaPy#103]
At present the `atomic` subpackage has a function called `charge_state` which returns an integer representing the charge (i.e., for an electron the charge state will be `-1` and for an alpha particle the charge state will be `2`).  It would be helpful to have a function called something like `electric_charge` in `atomic.py` that returns the actual charge as an Astropy `Quantity`.  This function will also need a docstring and tests.  It should have functionality like this:

```Python
>>> from plasmapy.constants import e
>>> from plasmapy.atomic import charge_state
>>> q = Z * charge_state('p')
<Quantity 1.6021766208e-19 C>
>>> electric_charge('p')
<Quantity 1.6021766208e-19 C>
```

Please post here if you decide to work on this, since I might be doing a bit of refactoring of `atomic` soon.  Thank you!

**Comment by nrb1324** [comment]
I'm taking a look at this. My python is very rusty so it might take a while.

**Comment by StanczakDominik** [comment]
Closed by #107.
